### ISSUE TITLE
Decorative element being read by screen readers as asterisk

### ISSUE THREAD

**Issue by rebekahmarkowitz** [WordPress/twentytwentyfour#41]
## Issue summary

Decorative element is reading as "asterisk" on screen readers

## Steps to reproduce
Implement voice over on macbook pro in chrome.

The decorative element in the below screen shot of the homepage is reading as "asterisk"

<img width="1336" alt="Screenshot 2023-08-24 at 3 06 20 PM" src="https://github.com/WordPress/twentytwentyfour/assets/93548879/ec45cdf5-6391-4461-8b14-973aefa92f2f">

## Recommendation
Asterisk should be an image with no alt text vs just text.

## Specifications
Template File: Homepage
**WCAG SC:** [1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
